soh/soh/z_message_OTR.cpp

```diff
@@ -187,4 +187,142 @@ extern "C" void OTRMessage_Init()
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_WITHDRAWAL_CONFIRM,
+        CustomMessage("Here are your rupees. Your new&balance is %y{{playerBalance}}%w rupees. See you&later!",
+                      "Hier sind deine Rupien. Ihr neues&Guthaben beträgt %y{{playerBalance}}%w Rupien.&Bis später!",
+                      "Voici vos roupies. Votre nouveau&solde est de %y{{playerBalance}}%w roupies. À plus&tard!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_DEPOSIT_AMOUNT,
+        CustomMessage("\x08Select Deposit Amount:&     rupees.&Set the amount with \xAA and press&\x9F to decide.\x0A",
+                    "\x08" "Einzahlungsbetrag:&     Rubine.&Wählen Sie den Betrag mit \xAA.&Drücken Sie \x9F, um zu entscheiden.\x0A",
+                    "\x08Sélectionnez le montant du dépôt:&     Roupies.&Réglez le montant avec \xAA et&appuyez sur \x9F pour décider.\x0A"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_DEPOSIT_CONFIRM,
+        CustomMessage("You've deposited your rupees. Your&new balance is %y{{playerBalance}}%w rupees. See&you later!",
+                    "Sie haben Ihre Rupien eingezahlt.&Ihr neues Guthaben beträgt %y{{playexrBalance}}%w&Rupien. Bis später!",
+                    "Vous avez déposé vos roupies.&Votre nouveau solde est de %y{{playerBalance}}%w&roupies. À plus tard!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_WARP_TRANSFER_INTRO,
+        CustomMessage("Ho ho! Your savings have reached&%y200%w rupees! A fine achievement,&young spender!",
+                    "Ho ho! Ihre Ersparnisse haben %y200%w&Rupien erreicht! Eine tolle Leistung,&junger Spender!",
+                    "Ho ho! Vos économies ont atteint&%y200%w roupies! Une belle réussite,&jeune dépensier!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_WARP_TRANSFER_ITEM,
+        CustomMessage("To reward your thriftiness, I've got&a special trinket for you...The&%rPirate's Charm%w!",
+                    "Um deine Sparsamkeit zu belohnen,&habe ich ein besonderes&Schmuckstück für dich... den&%rPiratenzauber%w!",
+                    "Pour récompenser votre économie,&j'ai un bijou spécial pour vous... Le&%rCharme du Pirate%w!"));
+    CustomMessageManager::Instance->CreateGetItemMessage(
+        //ITEM_RUPEE_BLUE is a placeholder for ITEM_PIRATE_CHARM which has not been implemented yet.
+        customMessageTableID, (GetItemID)TEXT_BANKER_REWARD_WARP_TRANSFER_LORE_1, ITEM_RUPEE_BLUE,
+        CustomMessage("This charm, infused with pirate&magic, can warp your treasures&directly to the bank! No more&heavy wallets for you!",
+                    "Dieser mit Piratenmagie&durchdrungene Zauber kann Ihre&Schätze direkt an die Bank bringen!&Keine schweren Geldbörsen mehr!",
```

Comment:
I suggest a different preposition for the German translation.

`an die Bank` &#8594; `in die Bank`

This alternative would be in accordance with the English text.

`an die Bank` &#8594; `zur Bank`